website/docs/sdk-reference/java.mdx

```diff
@@ -424,7 +424,7 @@ client.forceRefresh();
 
 With the following hooks you can subscribe to particular events fired by the SDK:
 
-- `onClientReady()`: This event is sent when the SDK reaches the ready state. If the SDK is configured with lazy load or manual polling it's considered ready right after instantiation.
+- `onClientReady(ClientReadyState)`: This event is sent when the SDK reaches the ready state. If the SDK is configured with lazy load or manual polling it's considered ready right after instantiation.
```

Comment:
```suggestion
- `onClientReady(ClientCacheState)`: This event is sent when the SDK reaches the ready state. If the SDK is configured with lazy load or manual polling it's considered ready right after instantiation.
```

[From what I see](https://github.com/configcat/java-sdk/blob/b36687881452567fbbe1a6d0e91e2e713c03f1ce/src/main/java/com/configcat/ConfigService.java#L72), "with lazy load or manual polling it's considered ready right after instantiation" is no longer true.

If so, I suggest a wording like [this](https://github.com/configcat/docs/blob/master/website/docs/sdk-reference/dotnet.mdx?plain=1#L421).